### ISSUE TITLE
[high] fix lists

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-shared-experiences#3]
e.g.,
* WCAG 2.0 Conformance Requirement: Conformance Requirement 4: Only Accessibility-Supported Ways of Using Technologies and
* Conformance Requirement 5: Non-Interference.
* WCAG 1.0 checkpoint: 6.3,
* 6.4,
* 6.5 and
* 8.1.

should be:
* WCAG 2.0 Conformance Requirement: Conformance Requirement 4: Only Accessibility-Supported Ways of Using Technologies and  Conformance Requirement 5: Non-Interference.
* WCAG 1.0 checkpoint: 6.3,  6.4, 6.5 and  8.1.

see https://www.w3.org/WAI/mobile/experiences

**Comment by yatil** [comment]
Yes, the content is not checked for rendering as noted in #1 (4th bullet). I am doubting that this is a correct use of lists, so probably major restructuring is needed. I’ll do that while linking the repository to the main site.
